SR2020/2019-11-29-still-interested.md

```diff
@@ -0,0 +1,12 @@
+---
+to: Teams that haven't responded to any emails
+subject: Still interested in SR2020?
+---
+
+Hello!
+
+Our records show that, other than an interest in taking part, we haven't heard anything from you this competition cycle. If you're still interested please reply to this email by the 9th December, with a signed kit disclaimer form and a shipping address we can send the kit to. If you don't respond we'll allocate your kit to another team.
```

Comment:
Maybe worth making it seem less forceful? 

```suggestion
Our records show that, other than an interest in taking part, we haven't heard anything from you this competition cycle. If you're still interested please reply to this email by the 9th December, ideally with a signed kit disclaimer form and a shipping address we can send the kit to. If you don't respond we'll be forced allocate your kit to another team.
```

Reply:
```suggestion
Our records show that, other than an interest in taking part, we haven't heard anything from you this competition cycle. If you're still interested please reply to this email by the 9th December, ideally with a signed kit disclaimer form and a shipping address we can send the kit to. If you don't respond we'll be forced to allocate your kit to another team.
```

Reply:
With regards to `ideally` is it really worth adding? If they don't send one we'll just need to send another email asking for it.